suite/tests/client-interface/cleancall.dll.c

```diff
@@ -600,6 +600,14 @@ static void
 app_exit_event(void)
 {
     check_stack_alignment();
+
+#if defined(LINUX) && defined(X86_64)
+    /* i#4335: Test allocation of more than 2.8GB in unreachable heap */
```

Comment:
I would say this fits better in the alloc.dll.c test as it is unrelated to clean calls?